test/ttmlir/Silicon/TTNN/emitc/two_fns.mlir

```diff
@@ -0,0 +1,16 @@
+// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline="system-desc-path=%system_desc_path%" %s > %t.mlir
```

Comment:
This PR work started as trying to fix the module op, which meant the removal of attrs and making sure that emitc works for multiple functions in a single module as well. This is why there's a seemingly random test included in this PR.